batchflow/research/research.py

```diff
@@ -468,16 +467,29 @@ class Executable:
         or returns (for function) values are lists of variable values
     path : str
         path to the folder where results will be dumped
-    exec : int, list of ints or None
+    execute : int, list of ints or None
```

Comment:
Does all this include @AlexeyKozhevin's research PR?

Reply:
It does not; all of the fixes are to get rid of `sphinx` warnings